reservation/app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\LoginRequest;
+use App\Http\Requests\RegisterRequest;
+
+use App\Repositories\AuthRepository;
+
+class AuthController extends Controller
+{
+    protected $authRepository;
+    public function __construct(AuthRepository $authRepository) {
+        $this->authRepository = $authRepository;
+        $this->middleware('auth:api', ['except' => ['login', 'register']]);
+    }
+
+    public function register(RegisterRequest $request) {
+        $data = [
+            "email" => $request->input('email'),
+            "name" => $request->input('name'),
+            "password" => bcrypt($request->input('password'))
```

Comment:
Nên cho phần này vào model User.php nhé
```suggestion
    public function setPasswordAttribute($value)
    {
        $this->attributes['password'] = bcrypt($value);
    }
```